clients/rust/build.rs

```diff
@@ -1,9 +1,71 @@
 extern crate protoc_rust;
 
+// This is a hack. protoc_rust depends on "protoc" being installed on
+// on the path. Without it, the docs.rs documentation build will fail
+// 
+// As far as I can tell there is no way to know if "cargo doc"
+// was called vs "cargo build". 
```

Comment:
I think you can use the `RUSTDOC` environment variable. https://doc.rust-lang.org/cargo/reference/environment-variables.html

Reply:
I tried and it it didn't work. I should probably open a issue.